ocrd_tesserocr/ocrd-tool.json

```
@@ -259,8 +259,8 @@
           "size": 1525436
         },
         {
-          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/frk.traineddata",
-          "name": "frk.traineddata",
+          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/deu_latf.traineddata",
+          "name": "deu_latf.traineddata",
           "parameter_usage": "without-extension",
           "description": "Tesseract LSTM model for historical (Fraktur typesetting and letterpress printing) German",
           "size": 6423052
```

Like I [said on your upstream change](https://github.com/tesseract-ocr/langdata_lstm/pull/59#issuecomment-1986832525), this now requires lots of downstream changes. (We used `frk` in various tests and workflows.)

Since it's too late to avoid breaking these use-cases, in the very least, this should be an alias, not a rename, on OCR-D's side.

```suggestion
        },
        {
          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/deu_latf.traineddata",
          "name": "frk.traineddata",
          "parameter_usage": "without-extension",
          "description": "Tesseract LSTM model for historical (Fraktur typesetting and letterpress printing) German",
          "size": 6423052
        },
        {
          "url": "https://github.com/tesseract-ocr/tessdata_fast/raw/main/deu_latf.traineddata",
          "name": "deu_latf.traineddata",
          "parameter_usage": "without-extension",
          "description": "Tesseract LSTM model for historical (Fraktur typesetting and letterpress printing) German",
          "size": 6423052
```

---

I added a resource entry for `frk` now, but marked it as deprecated in its description.

---

thanks!